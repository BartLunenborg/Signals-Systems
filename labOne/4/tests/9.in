1 100 100 3.23 0.4
