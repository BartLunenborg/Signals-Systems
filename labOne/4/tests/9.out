[92,-27,-98,2,99,22,-93,-46,82,66,-65,-83,44,94,-20,-99,-3,98,28,-91,-51,78,71,-60,-86,38,96,-14,-99,-10,97,34,-88,-56,74,75,-55,-89,33,97,-8,-99,-16,95,40,-85,-61,70,79,-50,-92,27,98,-2,-99,-22,93,46,-82,-66,65,83,-44,-94,20,99,3,-98,-28,91,51,-78,-71,60,86,-38,-96,14,99,10,-97,-34,88,56,-74,-75,55,89,-33,-97,8,99,16,-95,-40,85,61,-70,-79,50]
